jolpica/formula_one/data_correction.py

```diff
@@ -1,7 +1,12 @@
-from .models import SessionEntry, SessionType
+from .models import Session, SessionEntry, SessionType
 
 
 def run_data_correction():
```

Comment:
When changes are deployed is the data reimported every time with `make import-from-ergast`? The  [run_data_correction function](https://github.com/jolpica/jolpica-f1/blob/main/jolpica/formula_one/import_from_ergast.py#L964) runs in the import script so will require the import to be run again.

Reply:
This should be fine as the data being reverted each time is cause by it being incorrect in Ergast.
So we only need to correct it after an import.